### DISCUSSION TITLE
Relationship dynamic filter option picker

### DISCUSSION THREAD

**Discussion by uzemelteteshj** [Budibase/budibase#11572] [Help]
I have data table:
![image](https://github.com/Budibase/budibase/assets/113426887/b152901a-0828-4bca-96e9-70930acf3097)
I want to use in webapp if i chose 111 how option picker just "qewfdsf" and "sadasdasdas" but i cant setup filtering.

**Comment by ConorWebb96** [comment]
Hey @uzemelteteshj,

Have you tried using the relationship pick to setup the [dynamic filtering](https://docs.budibase.com/docs/filter-by-relationships)?

Could you provide a bit more information on what your goal is, as well as what you have tried?
